### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by rdshinde**
Added Modal Stylesheet and refactored colors file.

https://user-images.githubusercontent.com/67017632/153551318-792e60db-825e-4baf-96e9-b6c682947475.mov

### CODE REVIEW

File: components/modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const overlay = document.querySelector(".overlay");
+const modalAction = document.querySelectorAll(".modal__action");
+const modal = document.querySelectorAll(".modal");
+const openSimplemodal = document.querySelector(".open-simple-modal");
+
+const simplemodal = document.querySelector(".simple-modal");
```
Comment (shuntzhere) [inline components/modal/modal.js]: use openSimpleModal and simpleModal

**Review by shuntzhere — APPROVED** [review]
Looks good to me, just suggest a minor change.

File: components/modal/modal.js (hunk 1 above)
Comment (rdshinde) [reply components/modal/modal.js]: Ya , I will do it.

**Review by raman1999 — APPROVED** [review]
looks good to me, you can merge

**Pull request merged**